src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -183,6 +184,10 @@ public static function createConnection($dsn, array $options = [])
                 $host = $hosts[0]['host'] ?? $hosts[0]['path'];
                 $port = $hosts[0]['port'] ?? null;
 
+                if ($params['tls']) {
```

Comment:
beware, `?tls=false` might be casted to `true`

Reply:
Right, I added `$tls = filter_var($params['tls'], \FILTER_VALIDATE_BOOLEAN);`. Is it ok?